src/components/app/data/hooks/useIsBFFEnabled.js

```diff
@@ -0,0 +1,9 @@
+import { isBFFEnabled } from '../utils';
+import useEnterpriseCustomer from './useEnterpriseCustomer';
+import useEnterpriseFeatures from './useEnterpriseFeatures';
+
+export default function useIsBFFEnabled() {
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+  const { data: enterpriseFeatures } = useEnterpriseFeatures();
+  return isBFFEnabled(enterpriseCustomer.uuid, enterpriseFeatures);
```

Comment:
[nit/optional] I know tests already exist for the `isBFFEnabled` function, but because we have a new hook wrapping it, maybe just refactoring those tests to test for this hooks wrapper instead should pick this line up for coverage.

Reply:
Ah, yeah. Didn't see this was missing coverage. Will account for it in tests before merging. Thanks!